### ISSUE TITLE
How to hide grid when plotting a large matrix

### ISSUE THREAD

**Issue by vsimko** [taiyun/corrplot#42]
This might be a bug, inconsistent API, missing feature or bug in the documentation.

In `corrplot.mixed.R` the parameter `addgrid.col` is documented as:
``` r
#' @param addgrid.col The color of grid, if \code{NULL}, don't add grid.
```

In `corrplot.R` the parameter `addgrid.col` is documented as:
``` r
#' @param addgrid.col The color of grid. The default value is depends on
#'   \code{method}, if \code{method} is \code{color} or \code{shade}, the
#'   default values is \code{"white"}, otherwise \code{"grey"}.
```

It looks like it should be possible to use `addgrid.col` to disable the rendering of the grid.
But it does not work. See the following code snippet:
``` r
M <- matrix(runif(2500, 0.5, 1), nrow = 50)
corrplot(M, method = "color", cl.pos = "n", tl.pos = "n", addgrid.col = NULL)
```
![image](https://cloud.githubusercontent.com/assets/7080773/13776845/9eae710e-eaac-11e5-85f3-3f1948124947.png)

However, I would like to generate something like this:
![image](https://cloud.githubusercontent.com/assets/7080773/13776869/c2f9c34c-eaac-11e5-85a0-144296ded0a6.png)

**Comment by taiyun** [comment]
```
M <- matrix(runif(2500, 0.5, 1), nrow = 50)
corrplot(M, method = "color", cl.pos = "n", tl.pos = "n", addgrid.col = NA)
```

**Comment by vsimko** [comment]
Thanks, it works !
It means, that the `@param addgrid.col` documentation in `R/corrplot.mixed.R` and `R/corrplot.R` needs some update.

**Comment by taiyun** [comment]
YES. Many thanks for your passion and great job!

**Comment by vsimko** [comment]
It looks like that current API works as follows:
- `addgrid.col = NA` : no grid is rendered
- `addgrid.col = NULL` : chooses color based on the `method` parameter
- `addgrid.col = <string or number>` : renders the grid with user-defined color

**And I'm not sure if it is a feature or a bug.**

**Comment by taiyun** [comment]
YES.

It's a feature:)

**Comment by vsimko** [comment]
fixed in #47
